File: lib/screens/HomeScreen.java
import 'package:flutter/material.dart';
import 'package:flutter_svg/svg.dart';

class HomeScreen extends StatelessWidget {
  const HomeScreen({super.key});

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        backgroundColor: Colors.white,
        elevation: 2,
        shadowColor: const Color(0xffB6E388).withOpacity(1),
        title: SvgPicture.asset(
          "assets/images/logoGreen.svg",
          height: 30,
        ),
      ),
      body: const Center(
        child: Text("tlqkf"),
      ),
    );
  }
}

File: lib/screens/home_screen.dart
import 'package:flutter/material.dart';
import 'package:u_nil/screens/mypage_screen.dart';
import 'package:u_nil/screens/recipe_list_screen.dart';
import 'package:u_nil/widgets/bottomNavigation.dart';

class HomeScreen extends StatelessWidget {
  const HomeScreen({super.key});

  @override
  Widget build(BuildContext context) {
    return DefaultTabController(
      length: 4,
      child: Scaffold(
        backgroundColor: Colors.white,
        body: Column(
          children:  [
            Flexible(
              child: TabBarView(
                children: <Widget>[
                  const Text('1'),
                  const Text('2'),
                   RecipeListScreen(),
                  const MyPageScreen()
                ],
              ),
            ),
          ],
        ),
        bottomNavigationBar: const BottomNavigation(),
      ),
    );
  }
}
